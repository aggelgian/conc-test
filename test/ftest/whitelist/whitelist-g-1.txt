{whitelist,just_loop2,'*'}.
{lists,'*','*'}.
