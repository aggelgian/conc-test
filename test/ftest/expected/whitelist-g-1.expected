Testing whitelist:g/1 ...
INPUTS THAT LEAD TO RUNTIME ERRORS
#1 whitelist:g(4242)
